packages/node/src/indexer/dictionary.service.ts

```diff
@@ -239,34 +245,48 @@ export class DictionaryService implements OnApplicationShutdown {
     return buildQuery(vars, nodes);
   }
 
-  async getSpecVersion(): Promise<SpecVersion[]> {
+  async getSpecVersionMap(
```

Comment:
I find this function rather confusing. The naming doesn't make sense when comparing to `getSpecVersion` and functions should do only one thing where possible.

How about the interface below, then its easier to infer what the functions do

```ts

getSpecVersions(): SpecVersion[];
getSpecVersionsRaw(): SpecVersonDictionary);
parseSpecVersions(raw: SpecVersionDictionary): SpecVersion[];
```